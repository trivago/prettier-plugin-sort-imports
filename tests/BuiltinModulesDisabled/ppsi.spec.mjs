run_spec(__dirname, ['typescript'], {
    importOrder: ['^[./]'],
    importOrderParserPlugins: ['typescript']
});