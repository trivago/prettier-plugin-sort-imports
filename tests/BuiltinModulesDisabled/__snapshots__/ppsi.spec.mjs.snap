// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`builtin-modules.ts - typescript-verify > builtin-modules.ts 1`] = `
import express from 'express';
import lodash from 'lodash';
import fs from 'fs';
import path from 'path';
import http from 'http';
import { readFile } from 'node:fs';
import { join } from 'node:path';
import crypto from 'node:crypto';
import util from 'util';

import { myFunction } from './my-module';
import { anotherFunction } from '../other-module';~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import express from "express";
import fs from "fs";
import http from "http";
import lodash from "lodash";
import crypto from "node:crypto";
import { readFile } from "node:fs";
import { join } from "node:path";
import path from "path";
import util from "util";
import { anotherFunction } from "../other-module";
import { myFunction } from "./my-module";

`;
